alacritty/src/renderer/text/gles2.rs

```diff
@@ -0,0 +1,494 @@
+use std::mem::size_of;
+use std::ptr;
+
+use crossfont::RasterizedGlyph;
+use log::info;
+
+use alacritty_terminal::term::cell::Flags;
+use alacritty_terminal::term::SizeInfo;
+
+use crate::display::content::RenderableCell;
+use crate::gl;
+use crate::gl::types::*;
+use crate::renderer::shader::{ShaderProgram, ShaderVersion};
+use crate::renderer::{cstr, Error};
+
+use super::atlas::{Atlas, ATLAS_SIZE};
+use super::{
+    Glyph, GlyphCache, LoadGlyph, LoaderApi, TextRenderApi, TextRenderBatch, TextRenderer,
+};
+
+// Shader source.
+static TEXT_SHADER_F: &str = include_str!("../../../res/gles2/text.f.glsl");
+static TEXT_SHADER_V: &str = include_str!("../../../res/gles2/text.v.glsl");
+
+#[derive(Debug)]
+pub struct Gles2Renderer {
+    program: TextShaderProgram,
+    vao: GLuint,
+    vbo: GLuint,
+    ebo: GLuint,
+    atlas: Vec<Atlas>,
+    batch: Batch,
+    current_atlas: usize,
+    active_tex: GLuint,
+}
+
+impl Gles2Renderer {
+    pub fn new() -> Result<Self, Error> {
+        info!("Using OpenGL ES 2.0 renderer");
+
+        let program = TextShaderProgram::new(ShaderVersion::Gles2)?;
+        let mut vao: GLuint = 0;
+        let mut vbo: GLuint = 0;
+        let mut ebo: GLuint = 0;
+
+        let mut indices = Vec::with_capacity(BATCH_MAX / 4 * 6);
```

Comment:
Looks like these are vertex indices, right? If we have the space I'd probably go ahead and just name it `vertex_indices` just to make that obvious. Just `indices` isn't super clear on what their purpose is.